fuzzing/src/visitors/glyphvisitor.h

```diff
@@ -16,16 +16,14 @@
 #define VISITORS_GLYPH_VISITOR_H_
 
 
+#include "utils/noncopyable.h"
 #include "utils/utils.h"
 
-#include <boost/core/noncopyable.hpp>
-
-
 namespace freetype {
```

Comment:
💄 let's please make sure we keep two empty lines in here too

Reply:
Done.